risc0/zkvm/src/host/server/prove/exec.rs

```diff
@@ -245,8 +246,8 @@ impl CircuitStepHandler<Elem> for MachineContext {
 
     #[tracing::instrument(skip(self))]
     fn sort(&mut self, _: &str) {
-        self.memory.ram_plonk.sort();
-        self.memory.bytes_plonk.sort();
+        self.memory.memory_table.sort();
```

Comment:
Someone closer to the circuit code than me should check whether the various places you switched "RAM" to "Memory" in this PR are ok -- it's possible the "Random Access" portion of the term is load-bearing.

Reply:
oh good call -- i don't have any opinion between, e.g., `ram_table` vs. `memory_table`

Reply:
Many of these names have corresponding components in the circuit. In some places it's required that the names match (like in the case of extern calls that are made from the circuit out to the host). In other cases, it would be confusing to a circuit dev.

I really like the idea of renaming things, but we need to make the circuit changes first.

Reply:
Yeah I figured many of the changes I propose here will necessitate other changes in order to be coherent; I guess first we can sort out whether the suggestions I make here actually make sense, and if so, we can make a plan for how to get all the relevant dependencies sorted.

Reply:
I'm partial to `ram` rather than `memory` since `ram` is well known in CS and actually is more specific than memory.

Reply:
Updated to avoid making any changes `ram` and `memory`.